### PR TITLE
Use notistack instead of blueprint toaster

### PR DESCRIPTION
**Pull request by simonvadee**
I couldn't handle "oaster.show is not a function" one more single fucking time

### CODE REVIEW

File: client/src/app.tsx [hunk 1]
```diff
@@ -233,11 +234,25 @@ export const client = new ApolloClient({
   link: ApolloLink.from(links)
 });
 
+// add action to all snackbars
+const notistackRef = React.createRef<SnackbarProvider>();
```
Comment (BPierrick) [inline client/src/app.tsx:238]: I find it strange to initialize refs outside of component scope, but why not

File: client/src/components/newSource/index.tsx [hunk 2]
```diff
@@ -245,20 +216,16 @@ const NewSourceView = (): React.ReactElement => {
...
-        action: {
-          text: 'Interrompre'
-        },
-        uploadProgress: 0
-      })
-    );
+    const snack = enqueueSnackbar(`Creating source ${sourceName}...`, {
+      variant: 'info',
+      persist: true
+    });
```
Comment (Jasopaum) [inline client/src/components/newSource/index.tsx:228]: Oh no, the useless "Interrompre" button disappeared

File: client/src/components/mapping/ConceptMap/index.tsx [hunk 3]
```diff
@@ -515,14 +504,12 @@ const ConceptMapDialog = ({
           conceptMap
         );
       } catch (err) {
-        toaster.show({
-          message: `Could not update ConceptMap: ${
+        enqueueSnackbar(
+          `Could not create ConceptMap: ${
```
Comment (Jasopaum) [inline client/src/components/mapping/ConceptMap/index.tsx:508]: -> Could not update ConceptMap

File: client/src/app.tsx [hunk 4]
```diff
@@ -233,11 +234,25 @@ export const client = new ApolloClient({
...
 const App = (): React.ReactElement => (
   <Provider store={store}>
     <ApolloProvider client={client}>
       <PersistGate loading={null} persistor={persistor}>
-        <Routes />
+        <SnackbarProvider
+          ref={notistackRef}
+          action={key => (
+            <Button onClick={onClickDismiss(key)}>
+              <Icon icon="cross"></Icon>
```
Comment (Jasopaum) [inline client/src/app.tsx:250]: -> `<Icon icon="cross"/>`?

**Review by Jasopaum — APPROVED** [review]

File: client/src/app.tsx (hunk 1 above)
Comment (simonvadee) [reply client/src/app.tsx:238]: I used the examples from the doc!

**Pull request merged**